core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -460,4 +460,28 @@
    */
   boolean allowQualifyingCommonColumn();
 
+  /**
+   * Whether to allow the rewrite of the insert call in merge call.
+   *
+   * <p>For example, in the query
+   *
+   * <blockquote><pre>
+   * MERGE INTO t1 AS a
+   * USING t2 AS b
+   * ON a.x = b.x
+   * WHEN MATCHED THEN UPDATE SET
+   *   y = b.y
+   * WHEN NOT MATCHED THEN INSERT (x, y) VALUES (b.x, b.y)
+   * </pre></blockquote>
+   *
+   * <p>{@code VALUES (b.x, b.y)} is the inserted value. The default behavior
+   * is to rewrite it to {@code SELECT b.x, b.y FROM t2 AS b}. Which is not
+   * a valid grammar in BigQuery.
```

Comment:
nit: Should we replace "BigQuery" in this comment with "some dialects"?

Reply:
Fix in https://github.com/googleinterns/calcite/pull/139/commits/20a224efcc552eb936868ade31f0f9883ea844f6